src/test/java/seedu/duke/command/DeleteCommandTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.duke.command;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import seedu.duke.model.Item;
+import seedu.duke.model.ItemContainer;
+
+import java.math.BigDecimal;
+
+public class DeleteCommandTest {
+    private ItemContainer testList;
+    private Command testCommand;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        testList = new ItemContainer("test");
+        testList.addItem(new Item("HarryPotter", new BigDecimal("16.1"), new BigDecimal("25.12"), testList));
+        testCommand = new DeleteCommand("HarryPotter", testList);
+    }
+
+    @Test
+    public void deleteCommandExecute() {
```

Comment:
Test methods may follow the naming format e.g. methodBeingTested_inputGiven_expectedOutcome().

Reply:
Ok I will rename it. Thanks!